### PR TITLE
Fix update_project_team with Xcode 8

### PR DESCRIPTION
**Pull request by gcamp**
Thanks for contributing to _fastlane_! Before you submit your pull request, please make sure to check the following boxes:
- [x] Run `rspec` for all tools you modified (didn't have any spec)
- [x] Run `rubocop -a` to ensure the code style is valid
- [x] We currently don't accept new actions, please publish a plugin instead, more information in [Plugins.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Plugins.md)

Xcode now update DEVELOPMENT_TEAM in addition to DevelopmentTeam.

Fixes https://github.com/fastlane/fastlane/issues/6452

### CODE REVIEW

File: fastlane/lib/fastlane/actions/update_project_team.rb [hunk 1]
```diff
@@ -12,7 +12,9 @@ def self.run(params)
         UI.message("Updating development team (#{params[:teamid]}) for the given project '#{path}'")
 
         p = File.read(path)
-        File.write(path, p.gsub(/DevelopmentTeam = .*;/, "DevelopmentTeam = #{params[:teamid]};"))
+        p = p.gsub(/DevelopmentTeam = .*;/, "DevelopmentTeam = #{team_id};")
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/update_project_team.rb]: You can use

```ruby
p.gsub!(/DevelopmentTeam = .*;/, "DevelopmentTeam = #{team_id};")
```

instead 👍

**Review by KrauseFx — CHANGES_REQUESTED** [review]

**Review by KrauseFx — APPROVED** [review]

**Comment by KrauseFx** [comment]
I updated the pull request, this will be merged as soon as the build is green 👍

**Comment by gcamp** [comment]
Thanks for updating, I would have done it tomorrow!

**Comment by Pierre-Loup** [comment]
`params[:teamid]` has been replaced be `team_id` which rase the folowing error :

> [17:25:13]: Updating development team (RC2ZJG8GX8) for the given project '/Users/ptristant.ext/Developer/iOS/iOSRestructurationTydom/RestructurationTydom.xcodeproj/project.pbxproj'
[17:25:13]: Unknown method 'team_id'
[17:25:13]: Variable Dump:
[17:25:13]: {:PLATFORM_NAME=>nil, :LANE_NAME=>"delivery_appstore"}
[17:25:13]: To call another action from an action use `OtherAction.team_id` instead

**Comment by gcamp** [comment]
My bad, created a new pull request

**Pull request merged**